pyvlx/node_helper.py

```diff
@@ -1,15 +1,28 @@
 """Helper module for Node objects."""
+from typing import TYPE_CHECKING, Optional, Union
+
+from .api.frames import (
+    FrameGetAllNodesInformationNotification,
+    FrameGetNodeInformationNotification)
 from .const import NodeTypeWithSubtype
 from .lightening_device import Light
 from .log import PYVLXLOG
+from .node import Node
 from .on_off_switch import OnOffSwitch
 from .opening_device import (
     Awning, Blade, Blind, GarageDoor, Gate, RollerShutter, Window)
 
+if TYPE_CHECKING:
+    from pyvlx import PyVLX
+
 
-def convert_frame_to_node(pyvlx, frame):
+def convert_frame_to_node(pyvlx: "PyVLX",
+                          frame: Union[FrameGetNodeInformationNotification, FrameGetAllNodesInformationNotification]) -> Optional[Node]:
     """Convert FrameGet[All]Node[s]InformationNotification into Node object."""
     # pylint: disable=too-many-return-statements
+
+    assert frame.serial_number is not None
```

Comment:
I'm testing my PR #350 and at this point I have an issue, as serial number is None in many cases if just empty bytes are submitted by KLF200

Reply:
I'm fine with reverting this line. Now as you mention this, i recall, that some none-Velux devices may not have serial numbers.